develop/jsapi/add_feature_layer/lab.md

```diff
@@ -0,0 +1,54 @@
+###Add a feature layer to a map
+
+In this lab you will add a feature layer to an ArcGIS API for JavaScript application. 
+
+1. Click [create_starter_map/index.html](../create_starter_map/index.html) and copy the contents to a new [jsbin.com](http://jsbin.com).
+
+2. In `JSBin` > `HTML`, update the `require` statement and function definition:
+
+  ```javascript
+  require([
+    "esri/Map",
+    "esri/views/MapView",
+    "esri/layers/FeatureLayer", // ADD FeatureLayer module
```

Comment:
Same here. All of mine so far have had a few more imports so i put it on top instead of to the side.